sentry-samples/sentry-samples-android/build.gradle.kts

```
@@ -84,6 +84,20 @@ android {
         }
     }
 
+    flavorDimensions += "init"
+    productFlavors {
+        create("default") {
+            dimension = "init"
+            applicationIdSuffix = ".default"
+            versionNameSuffix = "-default"
+        }
+        create("early") {
+            dimension = "init"
+            applicationIdSuffix = ".early"
+            versionNameSuffix = "-early"
+        }
+    }
+
```

```suggestion
    flavorDimensions += "init"
    productFlavors {
        create("default") {
            dimension = "init"
            applicationIdSuffix = ".default"
            versionNameSuffix = "-default"
        }
        create("early") {
            dimension = "init"
            applicationIdSuffix = ".early"
            versionNameSuffix = "-early"
        }
    }

```

I'd rather remove the flavors since it makes the build slower, with more stuff to build and test, per flavor.